citest/gcp_testing/google_session.py

```diff
@@ -87,4 +99,6 @@ def cookies(self):
       session as a CookieJar object.
       CookieJar documentation: https://docs.python.org/2.7/library/cookielib.html#cookiejar-and-filecookiejar-objects
     """
+    if not self.__session.cookies:
```

Comment:
Done, it's theoretically possible to have zero cookies after login so we'll explicitly check it this way.